entity-management-web/src/main/java/eu/europeana/entitymanagement/web/EMController.java

```diff
@@ -54,12 +49,18 @@
     @Resource(name = AppConfig.BEAN_ENTITY_RECORD_SERVICE)
     private EntityRecordService entityRecordService;
 
+    @Resource(name = AppConfig.BEAN_ENTITY_RECORD_REPO)
+    private EntityRecordRepository entityRecordRepository;
```

Comment:
The controller must not access the repository directly, the required functionality must be accessible through the EntityRecordService. Please remove this reference to the repository

Reply:
Improved.